.changeset/tame-pens-occur.md

```diff
@@ -0,0 +1,6 @@
+---
+"@rocket.chat/meteor": patch
+"@rocket.chat/model-typings": patch
+---
+
+fix: Delete read receipts on messages deletion
```

Comment:
```suggestion
Deletes read receipts on messages deletion
```

Reply:
another option could be `Fixed read receipts not getting deleted after corresponding message is deleted` or something like that - but idon't think this has to be a "fix" changelog entry - could very well be a new thing for the change log. 

either way - would be better to not repeat the pr title and make it more changelog friendly (think users reading it)